specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/CommonDefinitions.json

```diff
@@ -101,6 +111,136 @@
         }
       }
     },
+    "ContainerAppProbe": {
+      "description": "Probe describes a health check to be performed against a container to determine whether it is alive or ready to receive traffic.",
+      "type": "object",
+      "properties": {
+        "exec": {
+          "description": "One and only one of the following should be specified. Exec specifies the action to take.",
```

Comment:
Wrong description?

Reply:
It sounded weird when I read this too. But this was pulled from the AKS OpenAPI Swagger so I left it as is. Its removed now since we're not supporting exec for this API version